src/main/java/services/ActionTypeStrategy.java

```diff
@@ -0,0 +1,7 @@
+package services;
+
+import services.actions.ActionHandler;
+
+public interface ActionTypeStrategy {
+    ActionHandler get(String type);
```

Comment:
This basically repeates purpose of the map, I believe we can remove this interface an impl